### ISSUE TITLE
Add grabbing files from pantheon to `composer site:import`

### ISSUE THREAD

**Issue by oliviaartz** [LastCallMedia/Drupal-Scaffold#68]
As it is this method grabs the database, but it leaves out all the image assets (and whatnot) referenced by the site.

**Comment by CybersecDan** [comment]
I feel like `composer site:import` shouldn't sync files as well, as the files aren't always necessary for local development. I'd be interested to hear what others think though.

**Comment by rbayliss** [comment]
I agree.  Ideally, we'd want to avoid grabbing files altogether by using something like stage file proxy.
